src/modules/feedback/components/FeedbackBox.js

```diff
@@ -73,9 +85,10 @@ export class FeedbackBox extends React.PureComponent<PropsType> {
           commentMessage={isPositiveRatingSelected ? t('positiveComment') : t('negativeComment')}
           onCommentChanged={onCommentChanged}
           required={!isPositiveRatingSelected} />
+        {errorMessage}
```

Comment:
Just inline this to get rid of evil `let`s :imp: :
```suggestion
        {feedbackSent === 'ERROR' && <Description>{t('failedSendingFeedback')}</Description>}
```